pkg/types/v1beta1/contants.go

```diff
@@ -17,7 +17,7 @@ limitations under the License.
 package v1beta1
 
 import (
-	"github.com/labring/sealos/pkg/utils/constants"
+	"github.com/labring/sealos/pkg/constants"
```

Comment:
The name of this file should be `constants.go` (Nothing wrong with the code here).